webapp-django/crashstats/api/views.py

```diff
@@ -92,29 +101,14 @@ def __init__(self, model, *args, **kwargs):
             self.fields[name] = field_class(required=required)
 
 
-# Names of models we don't want to serve at all
-API_DONT_SERVE_LIST = (
```

Comment:
I never liked this. It's far away from the things it's talking about so to create new API things, you have to make sure to edit this file in a different Django app otherwise your thing ends up public. That seems unhelpful.

Instead, I moved this idea to `IS_PUBLIC` which is a class attribute.